source/extensions/quic_listeners/quiche/envoy_quic_utils.cc

```diff
@@ -22,25 +22,30 @@ quicAddressToEnvoyAddressInstance(const quic::QuicSocketAddress& quic_address) {
              : nullptr;
 }
 
-quic::QuicSocketAddress envoyAddressInstanceToQuicSocketAddress(
-    const Network::Address::InstanceConstSharedPtr& envoy_address) {
-  ASSERT(envoy_address != nullptr && envoy_address->type() == Network::Address::Type::Ip);
-  uint32_t port = envoy_address->ip()->port();
+quic::QuicSocketAddress envoyAddressIpToQuicSocketAddress(const Network::Address::Ip* envoy_ip) {
```

Comment:
nit: rename to envoyIpAddressToQuicSocketAddress

Reply:
Renamed.